### ISSUE TITLE
Make possible to schedule articles to a later date

### ISSUE THREAD

**Issue by khaosdoctor** [forem/forem#6894]
**Is your feature request related to a problem? Please describe.**

It's not really related to a problem but to a pain I have. Most of the times I plan my month of articles and I have to start writing them in other apps before and leave them saved so I can post on the date, but the problem is that I sometimes am not at my computer or I have a trip scheduled and cannot publish the article.

**Describe the solution you'd like**

It'd be awesome if we could write articles on dev.to and then schedule them to be published on a later date.

**Describe alternatives you've considered**

I've considered creating some sort of app to post it to me but I haven't found any integration APIs.

**Comment by citizen428** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by citizen428** [comment]
@khaosdoctor Thanks for your suggestion! In the meantime, you can write and save your drafts on DEV instead of external tools.

1. Create post
2. Click "Save Draft"
3. Go to https://dev.to/dashboard
4. Select draft
5. Click edit and make any changes if necessary
6. Click publish

In case you want to further explore building your own tool, the relevant API documentation cna be found here:

* https://docs.dev.to/api/#section/Authentication
* https://docs.dev.to/api/#operation/createArticle

**Comment by khaosdoctor** [comment]
Awesome! Thanks a lot :)

**Comment by pkfrank** [comment]
Hi @khaosdoctor I also want to point out the [PublishTo.dev project](https://dev.to/toddanglin/publishto-dev-scheduling-article-publishing-on-dev-to-3m0o) developed by the community.

If you are interested in proposing a solution to how we could build this directly into the platform, we are open to your ideas and might green-light it for further community contributions.  We just want to scope out how users should interact with that functionality and how exactly it should work.

**Comment by khaosdoctor** [comment]
@pkfrank That's INCREDIBLE, this will most certainly solve my problems for now.

About the idea, let's see if I can better explain that, but I think it'd be easier to understand if I show you what other platforms have in place already.

So let's take Medium, when I draft a new story there's a `publish` button on top:
![image](https://user-images.githubusercontent.com/3200560/78700968-953a8500-78dc-11ea-983c-7cc1b7602c1b.png)

When you click on it there's an option to schedule:

![image](https://user-images.githubusercontent.com/3200560/78701656-c5ceee80-78dd-11ea-82bc-e8f3bff8d25e.png)

> I don't consider this to be a good UX, but there are other options

Then you just select day and time:

![image](https://user-images.githubusercontent.com/3200560/78701712-ded79f80-78dd-11ea-89b1-d515273cd8ba.png)

> This calendar is terrible, but it works

I don't know about technical implementation details, but I'm talking with Todd Anglin (the author of the publishToDev project) so we can implement an open API spec, this way Dev.to might be able to leverage the existing structure of the tool, just hosting it on its servers instead of externally.

**Comment by cmgorton** [comment]
Hello! Thanks for your contributions here.  I am going to close this issue.  We have a new process moving forward to address feature requests.
TLDR: we have new internal RFC process for feature requests. Any features that are more substantial and require more thought and detail will need to be discussed in [forem.dev](forem.dev).

You can read more about our Internal RFC Process and forem.dev Discussions in [this post](https://forem.dev/foremteam/internal-rfc-process-and-forem-dev-discussions-3gl4).
